### PR TITLE
initial pipeline clock support

### PR DESCRIPTION
**Pull request by aconchillo**

### CODE REVIEW

**Comment by LewisWolfgang** [comment]
Do we want to be replacing the cartesia example instead of just adding a new example for eleven labs?

File: src/pipecat/pipeline/task.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 from pydantic import BaseModel
 
+from pipecat.clocks.base_clock import BaseClock
```
Comment (LewisWolfgang) [inline src/pipecat/pipeline/task.py:13]: nit: this is unused, right?

Comment (aconchillo) [reply src/pipecat/pipeline/task.py:13]: No, we can pass a clock to the constructor. See below.

**Review by LewisWolfgang — APPROVED** [review]
lgtm

**Pull request merged**